### PR TITLE
Make the non-application-root-path consistent with policies

### PR DESCRIPTION
**Pull request by gwenneg**
Let's align with https://github.com/RedHatInsights/policies-ui-backend/pull/179, it will make things easier to maintain if all projects have the same behavior.

### CODE REVIEW

**Comment by gwenneg** [comment]
cc @pilhuhn

File: src/main/java/com/redhat/cloud/notifications/auth/RHIdentityAuthMechanism.java [hunk 1]
```diff
@@ -49,7 +49,7 @@
                     good = true;
                 }
             } else if (path.startsWith("/openapi.json") || path.startsWith("/internal")
-                    || path.startsWith("/admin") || path.startsWith("/health") || path.startsWith("/q/health")) {
+                    || path.startsWith("/admin") || path.startsWith("/health")) {
```
Comment (pilhuhn) [inline src/main/java/com/redhat/cloud/notifications/auth/RHIdentityAuthMechanism.java:52]: Can you add /metrics in here as well please?

Comment (gwenneg) [reply src/main/java/com/redhat/cloud/notifications/auth/RHIdentityAuthMechanism.java]: Sure, it's done.

**Comment by theute** [comment]
I'm merging.
@spinder that might impact you

**Pull request merged**